classes/api/forms.php

```diff
@@ -57,7 +82,36 @@ public function add_routes( $namespace ) {
 
 	}
 
+    /**
+     * Permissions for saving a form via REST API
+     * 
+     * Secures for POST /cf-api/v2/forms
+     * 
+     * @since 1.9.0
+     */
+	public function save_form_permissions_check(\WP_REST_Request $request){
+        //Is allowed?
+	    if( current_user_can( Caldera_Forms::get_manage_cap( 'manage' ) )  ){
+            //Allow if nonce valid
+	        return wp_verify_nonce( $request['cf_edit_nonce'],'cf_edit_element');
+        }
+	    return false;
+    }
 
+    /**
+     * Save a form via REST API
+     * 
+     * Handler for POST /cf-api/v2/forms
+     * 
+     * @since 1.9.0
+     */
+	public function save_form(\WP_REST_Request $request){
+        $saved = Caldera_Forms_Admin::save_a_form(array_merge(['ID' => $request['form_id']],$request['config']));
```

Comment:
Should we check if $request['form_id'] and $request['config'] are set?

Reply:
I don't think we need to beacuse:

- The request would not be valid without them, WordPress should refuse the request if they are not set.
- If they are not set correctly, ` Caldera_Forms_Admin::save_a_form` should return a WP_Error, which is expected on the next line.